packages/account/src/Sections/Security/ConnectedApps/connected-apps-info.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { InlineMessage, Text } from '@deriv/components';
+import { Localize } from '@deriv/translations';
+import { observer, useStore } from '@deriv/stores';
+import ConnectedAppsInfoBullets from './connected-apps-info-bullets';
+
+const ConnectedAppsInfo = observer(() => {
+    const { ui } = useStore();
+    const { is_mobile } = ui;
+
+    const text_size = is_mobile ? 'xxxs' : 'xxs';
+
+    return (
+        <InlineMessage
+            type='information'
+            size='md'
+            message={
+                <React.Fragment>
+                    <Text as='h4' weight='bold' size={text_size}>
+                        <Localize i18n_default_text='What are connected apps?' />
+                    </Text>
+                    <ConnectedAppsInfoBullets class_name_dynamic_suffix='with-apps' />
```

Comment:
**NIT:** Can the whole className that corresponds to the style be passed.  There may be repetition, but the code becomes easy to maintain